recipe/meta.yaml

```diff
@@ -507,6 +517,13 @@ outputs:
         - test ! -f $SP_DIR/functorch/__pycache__/__init__.cpython-313.pyc          # [py!=313 and unix]
         - if exist %SP_DIR%\functorch\__pycache__\__init__.cpython-313.pyc exit 1   # [py!=313 and win]
 
+        # test integrity of CMake metadata
```

Comment:
This comment could be extended, like "and that <...> is visible as a symbol from libtorch_python".